routes/account.js

```diff
@@ -491,6 +494,11 @@ module.exports = function (
         db.account(Buffer(uid, 'hex'))
           .then(
             function (account) {
+              // If the account is already verified, they may be e.g.
+              // clicking a stale link.  Silently succeed.
+              if (account.emailVerified) {
+                return true
+              }
```

Comment:
Is this related to the locked account changes or just something that you saw should be there? Am just checking to see if it's part of it or some additional bits (am happy if it is, just wondering).